Shared/Samples/Authenticate with Integrated Windows Authentication/AuthenticateWithIntegratedWindowsAuthenticationView.swift

```diff
@@ -0,0 +1,189 @@
+                    }
+                    .buttonStyle(.plain)
+                }
+            case .failure:
+                urlEntryView
+                ContentUnavailableView(
+                    "Error",
+                    systemImage: "exclamationmark.triangle",
+                    description: Text("Error searching specified portal.")
+                )
+            case nil:
+                if model.isConnecting {
+                    ProgressView()
+                        .frame(maxWidth: .infinity)
+                } else {
+                    urlEntryView
+                }
+            }
+        }
+        .onDisappear {
+            Task {
+                // Reset the challenge handlers and clear credentials
+                // when the view disappears so that user is prompted to enter
+                // credentials every time the sample is run, and to clean
+                // the environment for other samples.
+                await model.teardownAuthenticator()
+            }
+        }
+        .sheet(item: $model.selectedItem) { selectedItem in
+            NavigationStack {
+                MapView(map: Map(item: selectedItem.portalItem))
+                    .navigationTitle(selectedItem.portalItem.title)
+                    .navigationBarTitleDisplayMode(.inline)
+                    .toolbar {
+                        ToolbarItem(placement: .topBarTrailing) {
+                            Button("Done") {
+                                model.selectedItem = nil
+                            }
+                        }
+                    }
+            }
+            .interactiveDismissDisabled()
+            .highPriorityGesture(DragGesture())
+            .pagePresentation()
+        }
+        .animation(.default, value: model.isConnecting)
+        .authenticator(model.authenticator)
+    }
+    
+    @ViewBuilder private var urlEntryView: some View {
+        Section {
+            HStack {
+                TextField("IWA Secured Portal URL", text: $model.portalURLString)
+                    .onSubmit { Task { await model.connectToPortal() } }
+                    .autocapitalization(.none)
+                    .keyboardType(.URL)
+                Button("Connect") {
+                    Task { await model.connectToPortal() }
+                }
+                .disabled(model.portalURL == nil)
+            }
+        }
+    }
+}
+
+/// A value that represents an item selected by the user.
+private struct SelectedItem: Identifiable {
+    /// The portal item that was selected.
+    let portalItem: PortalItem
+    
+    var id: ObjectIdentifier {
+        ObjectIdentifier(portalItem)
+    }
+}
+
+extension AuthenticateWithIntegratedWindowsAuthenticationView {
+    @MainActor
+    class Model: ObservableObject {
+        /// The authenticator to handle authentication challenges.
+        let authenticator = Authenticator(promptForUntrustedHosts: true)
```

Comment:
The `promptForUntrustedHosts` should not be in the final sample. The project does have `NSAllowsArbitraryLoads: true` so untrusted prompt will not reach anyways.
```suggestion
        let authenticator = Authenticator()
```